<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_edit</class>
 <widget class="QDialog" name="player_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos del jugador</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>611</width>
     <height>465</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_nombre"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_apellido"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fecha Nacimiento:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="date_fecha_nac">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Lugar Nacimiento:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line_lugar_nac">
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Provincia:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>DNI:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="line_dni">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Domicilio:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="line_domicilio"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Fecha Inscripción:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDateEdit" name="date_fecha_incripcion">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>División:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="pin_division"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Observaciones:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QTextEdit" name="txt_observaciones"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Fecha de actualización:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="lbl_fecha_renovacion">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Vigencia hasta:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="lbl_fecha_vigencia">
        <property name="font">
         <font>
          <family>Arial Sans</family>
          <pointsize>14</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Arial Sans&quot;;
color: rgb(85, 0, 0);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="line_provincia"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Club:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cmb_clubes"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_photo_player">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/asserts/default-user.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_change_photo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 0);</string>
        </property>
        <property name="text">
         <string>Cambiar foto...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_print_card">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>Imprimir credencial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_nombre</tabstop>
  <tabstop>line_apellido</tabstop>
  <tabstop>date_fecha_nac</tabstop>
  <tabstop>line_lugar_nac</tabstop>
  <tabstop>line_provincia</tabstop>
  <tabstop>line_dni</tabstop>
  <tabstop>line_domicilio</tabstop>
  <tabstop>date_fecha_incripcion</tabstop>
  <tabstop>pin_division</tabstop>
  <tabstop>cmb_clubes</tabstop>
  <tabstop>txt_observaciones</tabstop>
  <tabstop>btn_change_photo</tabstop>
  <tabstop>btn_save</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>btn_print_card</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
